pkg/codegen/nodejs/doc.go

```diff
@@ -150,10 +150,14 @@ func (d DocLanguageHelper) GetEnumName(e *schema.Enum, typeName string) (string,
 func (d DocLanguageHelper) GetModuleDocLink(pkg *schema.Package, modName string) (string, string) {
 	var displayName string
 	var link string
+	namespace := "@pulumi"
+	if pkg.Namespace != "" {
+		namespace = "@" + pkg.Namespace
+	}
 	if modName == "" {
-		displayName = "@pulumi/" + pkg.Name
+		displayName += namespace + "/" + pkg.Name
```

Comment:
```suggestion
		displayName = namespace + "/" + pkg.Name
```

Nits - Why "+="? Also, maybe use fmt.Sprintf for consistency?